CHANGELOG.md

```diff
@@ -3,6 +3,9 @@ All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
+## [2.3.1] - 2017-10-19
```

Comment:
we actually bump the version and date it on merge and release on the maintainers side. https://github.com/sensu-plugins/community/blob/master/HOW_WE_CHANGELOG.md#unreleased-change as there is no guarantee on merge order, when we will get to it, etc. Also this is a `Breaking Change` so we would bump it a major version.